### DISCUSSION TITLE
Spreading the word to the Italian market

### DISCUSSION THREAD

**Discussion by valerio-afk** [TodePond/GulfOfMexico#391] [Show and tell]
The recently renamed DreamBerd must be known far and wide.
For this reason, I am spreading the word in my home country, Italy, although I happen to live in the UK (and Italy has not been renamed into the UK, yet!).

Although the video is in italian, you may enjoy it with English subs: https://youtu.be/aM5hMPg0DDc

**Comment by TodePond** [comment]
Hey that's awesome! you went into detail! I haven't watched the whole thing yet but will finish later and share around :)
